test/actions/recover.js

```diff
@@ -0,0 +1,225 @@
+    const dispatchedActions = store.getActions();
+    const dispatchedActionTypes = dispatchedActions.map(getActionType);
+    const expectedActionTypes = [
+      'COLLECTION_INFLIGHT',
+      'COLLECTION',
+      'COLLECTION_APPLYWORKFLOW_ERROR'
+    ];
+    t.deepEqual(expectedActionTypes, dispatchedActionTypes);
+  });
+});
+
+test.serial('applyRecoveryWorkflowToCollection successfully sends applyWorkflow action', async (t) => {
+  const getCollectionResponse = {
+    results: [{
+      meta: {
+        collectionRecoveryWorkflow: 'someWorkflow'
+      }
+    }]
+  };
+
+  nock('https://example.com')
+    .get(`/collections?name=${name}&version=${version}`)
+    .reply(200, getCollectionResponse);
+  nock('https://example.com')
+    .put(`/collections/${name}/${version}`)
+    .reply(200);
+
+  return store.dispatch(applyRecoveryWorkflowToCollection(collectionId)).then(() => {
+    const dispatchedActions = store.getActions();
+    const dispatchedActionTypes = dispatchedActions.map(getActionType);
+    const expectedActionTypes = [
+      'COLLECTION_INFLIGHT',
+      'COLLECTION',
+      'COLLECTION_APPLYWORKFLOW_INFLIGHT',
+      'COLLECTION_APPLYWORKFLOW'
+    ];
+    t.deepEqual(expectedActionTypes, dispatchedActionTypes);
+  });
+});
+
+test.serial('applyRecoveryWorkflowToGranule fails to acquire granule, dispatches GRANULE_APPLYWORKFLOW_ERROR', async (t) => {
+  nock('https://example.com')
+    .get(`/granules/${granuleId}`)
+    .reply(404);
+
+  return store.dispatch(applyRecoveryWorkflowToGranule(granuleId)).then(() => {
+    const dispatchedActions = store.getActions();
+    const dispatchedActionTypes = dispatchedActions.map(getActionType);
+    const expectedActionTypes = [
+      'GRANULE_INFLIGHT',
+      'GRANULE_ERROR',
+      'GRANULE_APPLYWORKFLOW_ERROR'
+    ];
+    t.deepEqual(expectedActionTypes, dispatchedActionTypes);
+  });
+});
+
+test.serial('applyRecoveryWorkflowToGranule related collection does not have recoveryWorkflow configured', async (t) => {
+  const getGranuleResponse = {
+    collectionId
+  };
+  const getCollectionResponse = {
+    results: [{
+      meta: {}
+    }]
+  };
+
+  nock('https://example.com')
+    .get(`/granules/${granuleId}`)
+    .reply(200, getGranuleResponse);
+  nock('https://example.com')
+    .get(`/collections?name=${name}&version=${version}`)
+    .reply(200, getCollectionResponse);
+
+  return store.dispatch(applyRecoveryWorkflowToGranule(granuleId)).then(() => {
+    const dispatchedActions = store.getActions();
+    const dispatchedActionTypes = dispatchedActions.map(getActionType);
+    const expectedActionTypes = [
+      'GRANULE_INFLIGHT',
+      'GRANULE',
+      'COLLECTION_INFLIGHT',
+      'COLLECTION',
+      'GRANULE_APPLYWORKFLOW_ERROR'
+    ];
+    t.deepEqual(expectedActionTypes, dispatchedActionTypes);
+  });
+});
+
+test.serial('applyRecoveryWorkflowToGranule fails to acquire collection, fails, and dispatches failure', async (t) => {
```

Comment:
```suggestion
test.serial('applyRecoveryWorkflowToGranule fails to acquire collection and dispatches GRANULE_APPLYWORKFLOW_ERROR', async (t) => {
```

I don't actually feel strongly about this, but however we name the tests we should be consistent